npm-shrinkwrap.json

```diff
@@ -0,0 +1,5691 @@
+{
```

Comment:
I think for now, considering that we mostly don't even keep package-lock.json around, I'm inclined to skip npm-shrinkwrap.json for now.

I think it's something we should have a larger conversation around, and possibly make an RFD for.